charts/rating/values.yaml

```diff
@@ -70,6 +70,10 @@ tolerations: []
 
 affinity: {}
 
+#flag to disable readinessProbe in deployment.yaml L38
+readinessProbe:
+  enabled: false
```

Comment:
```suggestion
  enabled: true
```
This should be enabled by default. Disabling probes should be a voluntary act :)